content/en/agent/guide/private-link.md

```diff
@@ -28,75 +24,166 @@ The overall process consists of configuring an internal endpoint in your VPC tha
+
+9. Once it shows _Available_, the AWS PrivateLink is ready to be used.
+
+### Amazon Route53
+
+1. Create a [Route53 private hosted zone][2] for each service for which you have created an AWS PrivateLink endpoint. Attach the private hosted zone to the VPC in `us-east-1`.
+
+    You can find this information by referencing the table below or interrogating the AWS APIs, `DescribeVpcEndpointServices`, or by using the following CLI command: `aws ec2 describe-vpc-endpoint-services --service-names <service-name>`. For example, in the case of the Datadog metrics endpoint:
+
+    ```bash
+    aws ec2 describe-vpc-endpoint-services --service-names com.amazonaws.vpce.us-east-1.vpce-svc-09a8006e245d1e7b8 | jq '.ServiceDetails[0].PrivateDnsName'
+    ```
+
+    This returns `metrics.agent.datadoghq.com`. This is the name of the private hosted zone that you need to create and associate with the VPC which the Agent traffic originates in. Overriding this record grabs all dynamic Agent-versioned hostnames that exist.
 
-    Once it shows _Available_, the AWS PrivateLink is ready to be used.
-11. If you are collecting logs data, ensure your Agent is configured to send logs over HTTPS. If the data is not already there, add the following to the [Agent `datadog.yaml` configuration file][3]:
+    Use the list below to map service and DNS name to different parts of Datadog:
+
+    | Datadog             | PrivateLink service name                                  | Private DNS name                                   |
+    |---------------------| --------------------------------------------------------- | -------------------------------------------------- |
+    | Metrics             | `com.amazonaws.vpce.us-east-1.vpce-svc-09a8006e245d1e7b8` | `metrics.agent.datadoghq.com`                     |
+    | Logs (HTTP intake)  | `com.amazonaws.vpce.us-east-1.vpce-svc-025a56b9187ac1f63` | `agent-http-intake.logs.datadoghq.com`            |
+    | API                 | `com.amazonaws.vpce.us-east-1.vpce-svc-064ea718f8d0ead77` | `api.datadoghq.com`                               |
+    | Process             | `com.amazonaws.vpce.us-east-1.vpce-svc-0ed1f789ac6b0bde1` | `process.datadoghq.com`                           |
+    | Traces              | `com.amazonaws.vpce.us-east-1.vpce-svc-0355bb1880dfa09c2` | `trace.agent.datadoghq.com`                       |
+    | Containers          | `com.amazonaws.vpce.us-east-1.vpce-svc-0ad5fb9e71f85fe99` | `orchestrator.datadoghq.com`                      |
+
+{{< img src="agent/guide/private_link/create-a-route53-private-hosted-zone.png" alt="Create a Route53 private hosted zone" style="width:80%;" >}}
+
+2. Within each new Route53 private hosted zone, create an A record with the same name. Toggle the **Alias** option, then under **Route traffic to**, choose **Alias to VPC endpoint**, **us-east-1**, and enter the DNS name of the VPC endpoint associated with the DNS name.
```

Comment:
Yes, this would definitely be helpful -- whether in screenshot or through AWS docs. Do you know a specific instance where this is mentioned? If not, I can do some digging and check.